src/exchange/group/delegation/group-delegation.controller.js

```diff
@@ -146,5 +156,40 @@ angular.module('Module.exchange.controllers').controller(
           this.services.navigation.resetAction();
         });
     }
+
+    getDefaultDomain() {
+      const name = _.last(this.currentAccount.split('@'))
+      return {
+        displayName: name,
+        name: name
+      };
+    }
+
+    fetchAccountCreationOptions() {
+      this.loadingDomains = true;
+      return this.services.Exchange.fetchingAccountCreationOptions(
+        this.$routerParams.organization,
+        this.$routerParams.productId,
+      )
+        .then((accountCreationOptions) => {
+          const domains = [this.allDomainsOption];
+          _.forEach(accountCreationOptions.availableDomains, domain => {
```

Comment:
You will ended up with an ESLint `arrow-parens` error.
```suggestion
          _.forEach(accountCreationOptions.availableDomains, (domain) => {
```